lib/plugins/dev-server.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const pathUtil = require('../config/path-util');
+const AssetOutputDisplayPlugin = require('../friendly-errors/asset-output-display-plugin');
+const FriendlyErrorsPlugin = require('./friendly-errors');
+
+/**
+ * @param {WebpackConfig} webpackConfig
+ * @param {Array} paths to clean
+ * @param {Object} cleanUpOptions
+ * @return {Array} of plugins to add to webpack
+ */
+module.exports = {
+    getPlugins(webpackConfig) {
+        const outputPath = pathUtil.getRelativeOutputPath(webpackConfig);
+        const friendlyErrorsPlugin = FriendlyErrorsPlugin.getPlugins();
+        return [new AssetOutputDisplayPlugin(outputPath, friendlyErrorsPlugin[0])];
```

Comment:
> This case looks to me like a good scenario for a singleton

Yes.... but no :). We allow you to generate multiple webpack configs. And to be strict about it, each separate webpack config should have its own instances of plugins. 

So, yes, we need to make sure that we don't create multiple `FriendlyErrorsWebpackPlugin` while generating *one* webpack config... but *do* need to create distinct instances, each time we `ConfigGenerator.getWebpackConfig()`. If you can take a shot at that, perfect :).